{"Launchpad":"https://bugs.launchpad.net/bugs/1404801","Id":"1404801","Gerrit":"https://review.openstack.org/145725","IdGerrit":"145725","IdCommit":"2dec248dc767ed0a9e064c3a61429387b5740bac","Files":["nova/conductor/manager.py","nova/tests/conductor/test_conductor.py"],"Lines_Inseted":19,"Lines_Deleted":12,"CommitParent":"fb2570da3687ec0ab9baefdf9eabf5ab11e7cd57","Classification":"Bug","KeywordsTitle":" instance not working if instance is boot from volume","Description":" it fails on the conductor with erroR, ","KeywordsCommit":".","Comments":"Malfunctionality"}